src/tools/index.js

```diff
@@ -0,0 +1,19 @@
+import path from 'path';
+import simpleGit from 'simple-git';
+
+export async function getModifiedServices(rootPath) {
```

Comment:
As the path for `services` is always the same no matter where we call this function, I'm in favor of not passing `rootPath` as parameter but defining it directly inside this module.